extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -129,6 +129,11 @@ export class AmpAudio extends AMP.BaseElement {
       artwork: [{src: artwork}],
     };
 
+    // Resolve layoutCallback right away if the audio won't preload.
+    if (this.element.getAttribute('preload') === 'none') {
```

Comment:
I'm not sure I understand this change. Can you explain what the issue is?

Reply:
The `preload` attribute set to `none` will delay the loading of the video/audio until user clicks play. The `loadPromise` would be pending until then, preventing the `layoutCallback` from resolving.
TLDR: this change ensures the `layoutCallback` doesn't wait for a user to click "play" to resolve.